### PR TITLE
Heading changes + slight copy edits

### PR DESCRIPTION
**Pull request by juandolealt**

### CODE REVIEW

File: pages/faucet/solana.tsx [hunk 1]
```diff
@@ -30,9 +30,9 @@ const SolanaFaucet: ThirdwebNextPage = () => {
         mx="auto"
         px={{ base: 0, md: 4 }}
       >
-        <Heading>Solana faucet</Heading>
-        <Heading fontSize="20px" my="4">
-          Get Solana devnet tokens for free
+        <Heading mr={3} as="h1">Solana Faucet</Heading>
```
Comment (jnsdls) [inline pages/faucet/solana.tsx:33]: for the extra spacing here `mr={3}` let's instead add `gap={3}` on the `<Flex />` component above (prefer gap spacing over margin spacing)

**Review by jnsdls — COMMENTED** [review]
just a quick improvement

File: pages/faucet/solana.tsx (hunk 1 above)
Comment (nachoiacovino) [reply pages/faucet/solana.tsx:33]: the mr was a copy/paste mistake as Juan told me, he'll fix it

**Comment by jnsdls** [comment]
did a quick fix in #610, want to get this in now

**Pull request closed without merging**